runtime-network/aml/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.3.0
+SRCS="https://github.com/any1/aml/archive/refs/tags/v$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.